config/config.go

```diff
@@ -100,6 +129,7 @@ func DefaultInitOpts() InitOpts {
 		MaxFileSize:       DefaultMaxFileSize,
 		ComputeProviderID: BestProviderID,
 		Throttle:          false,
+		Scrypt:            DefaultLabelsScryptParams(),
```

Comment:
NIT: same here, instead of `Scrypt` I would call it `LabelParams` or `InitParams`.

Reply:
I'm not sure about this one. The structure gathers label initialization options. This information is already included in its name. It's sort of obvious that the options that it contains control label initialization. The field name `Scrypt` informs the reader that this field controls the scripting part of it.

Reply:
I mean `DefaultLabelParams()` instead of `DefaultLabelsScryptParams()`